src/new-client/src/App/Trades/TradesFooter.tsx

```diff
@@ -14,6 +14,11 @@ const TradesFooterStyled = styled("div")`
   color: ${({ theme }) => theme.core.textColor};
   background-color: ${({ theme }) => theme.core.lightBackground};
   border-radius: 0 0 0.25rem 0.25rem;
+  margin-top: 5px;
```

Comment:
Really minor - think it makes sense to give top/bottom/left/right values for position absolute elements, rather than margins.

Reply:
Fixed